src/pybricks/hubs.py

```diff
@@ -38,6 +41,25 @@ class MoveHub:
     imu = _common.SimpleAccelerometer()
     system = _common.System()
     button = _common.Keypad([_Button.CENTER])
+    ble = _common.BLE()
+
+    def __init__(
+        self, broadcast_channel: int = 0, observe_channels: Sequence[int] = []
```

Comment:
Shouldn't the broadcast channel be something like `None` to be consistent with an empty set of observe channels?

Reply:
Since we don't start broadcasting until `hub.ble.broadcast()` is called, it doesn't really matter. By having a default channel it will "just work" even if a user forgets to add the arg.

Reply:
Right, I was thinking along the lines of "explicit is better than implicit" (we require the channel number everywhere else).

With a zero default, I think people might accidentally end up with two hubs broadcasting at `0` and wondering why performance isn't great.